ecs-agent/tmds/handlers/v1/credentials_handler.go

```diff
@@ -144,8 +144,8 @@ func processCredentialsRequest(
 	credentialsJSON, err := json.Marshal(credentials.IAMRoleCredentials)
 	if err != nil {
 		errText := errPrefix + "Error marshaling credentials"
-		seelog.Errorf("Error processing credential request credentialType=%s taskARN=%s: %s",
-			credentials.IAMRoleCredentials.RoleType, credentials.ARN, errText)
+		seelog.Errorf("Error processing credential request credentialType=%s taskARN=%s credentialScope=%s: %s",
```

Comment:
nit: I know using seelog is the existing behavior here, but typically we prefer to use the structured logger defined in `github.com/aws/amazon-ecs-agent/ecs-agent/logger` for logging in ecs-agent module. Could we use the structured logger for logging in this file to follow that convention and help make the logs more readable and traceable?

Example for reference: https://github.com/aws/amazon-ecs-agent/pull/3696/files#diff-b75f163bbc1723848662c8e77f544e15ebac5045354e2a4af8b057b502523252L386-R389

Reply:
Sure! I will update it

Reply:
Synced offline, @Yiyuanzzz will switch to use structured logger in a followup pull request as this is non-blocking to this pull request.